pkg/yurthub/proxy/proxy.go

```diff
@@ -98,7 +102,20 @@ func (p *yurtReverseProxy) buildHandlerChain(handler http.Handler) http.Handler
 }
 
 func (p *yurtReverseProxy) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
-	if !hubutil.IsKubeletLeaseReq(req) && p.loadBalancer.IsHealthy() {
+	now := time.Now()
+	isKubeletLeaseReq := hubutil.IsKubeletLeaseReq(req)
+	if isKubeletLeaseReq {
+		p.loadBalancer.Resume()
+		p.lastKubeletLeaseReqTime = now
+	}
+
+	if !p.lastKubeletLeaseReqTime.IsZero() && p.healthCheckGracePeriod > 0 {
+		if now.After(p.lastKubeletLeaseReqTime.Add(p.healthCheckGracePeriod)) {
+			p.loadBalancer.Pause()
```

Comment:
we can not use lease request or other http requests to trigger to pause healthyChecker, because no requests except watch requests can be sent to yurthub when kubelet stopped.

btw: maybe we only need to update `lastKubeletLeaseReqTime` here, and then check kubelet stopped or not in HealthyChecker `sync` interval.